api/src/main/resources/liquibase.xml

```diff
@@ -567,6 +567,33 @@
         </sql>
 
     </changeSet>
+    <changeSet id="add_visit_date_patient_appointment_20230222" author="makombe">
```

Comment:
If we are merging this in from a PR, hopefully we can just merge this down into one changeSet that adds the right column with the right name.